_layouts/py1-lesson.html

```diff
@@ -0,0 +1,76 @@
+---
+layout: default
+fw: true
+---
+<div class="py1-lesson-page-wrapper">
+    <div class="py1-lesson-sidebar">
+        <div class="py1-section1">
+            <h1 class="py1-lesson-header"> {{ page.num }}. {{ page.title }} </h1>
+            <p class="py1-lesson-page-description">
+                {{ content }}
+            </p>
+
+            <p class="py1-lesson-page-agenda">
+                <b> agenda </b>
+                {{ page.agenda }}
+            </p>
+
+            <div class="py1-lesson-page-action-bar">
+                {% if page.slides_link %}
+                <a href="{{ page.slides_link }}" target="_blank">slides</a>
+                {% endif %}
+
+                {% if page.worksheet_link %}
+                <a href="{{ page.worksheet_link }}" target="_blank">worksheet</a>
+                {% endif %}
+
+                <span class="py1-lesson-duration">{{ page.length }} min</span>
+            </div>
```

Comment:
If we're duplicating the styling of the card from the main page:

* do we want to add the border?
* do we want to abstract this to an `_include`?